packages/grafana-data/src/transformations/standardTransformersRegistry.ts

```diff
@@ -28,6 +28,26 @@ export interface TransformerRegistryItem<TOptions> extends RegistryItem {
    * React component used as UI for the transformer
    */
   editor: React.ComponentType<TransformerUIProps<TOptions>>;
+
+  /**
+   * Set of categories associated with the transformer
+   */
+  categories?: Set<TransformerCategory>;
+
+  /**
+   * Image path for the transformer
+   */
+  image?: { dark: string; light: string };
```

Comment:
We can just use the id, and a naming pattern to get the image path

Reply:
theme (dark or light) is a runtime thing, but image path is a build time thing.

So we need to import both versions anyway, for the frontend build system to pick them up, optimize and create an output path with a hash in the name of the file (that we don't know of before end).

What can I do is set the "image" attribute type to be string, and assign the right image at runtime.

Reply:
I can see that assets in the "public/img" folder are generally used without including them in the build process. So there is no optimization and no hash on filename, which is usefull for browser caching efficiency.

Do you advice to also does the same for the transformations illustration?

Reply:
Did it the way you described